src/Magnum/Platform/GlfwApplication.cpp

```diff
@@ -403,8 +403,12 @@ bool GlfwApplication::tryCreate(const Configuration& configuration, const GLConf
 
     /* Create window. Hide it by default so we don't have distracting window
        blinking in case we have to destroy it again right away. If the creation
-       succeeds, make the context current so we can query GL_VENDOR below. */
-    glfwWindowHint(GLFW_VISIBLE, false);
+       succeeds, make the context current so we can query GL_VENDOR below.
+       If we are on Wayland, this is causing a segfault; a blinking window is
+       acceptable in this case. */
+    constexpr static const char waylandString[] = "wayland";
```

Comment:
```suggestion
    constexpr const char waylandString[] = "wayland";
```

I have this bad stigma of function-local statics being wrapped in mutexes and `std::call_once` stuff, so trying to not use it to avoid raising a red flag whenever I see the code :D If I'm right, `constexpr const char` is put into readonly memory anyway, so it shouldn't have any perf disadvantages.